src/samples/audio/nxp_detect_test.c

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright 2021 NXP
+//
+// Author: Cristina Feies <cristina.ilie@nxp.com>
+// Author: Viorel Suman <viorel.suman@nxp.com>
```

Comment:
C-style comments... I think. See what Checkpatch says about this.

Reply:
While not making much difference in this case, C99 comments are very useful for trailing comments and they're 12 years old now! I think they should be allowed.

Reply:
@paulstelian97 This is OK, SPDX uses C++-style comments for normal files and C-style comments for headers:

https://lwn.net/Articles/739183/

Reply:
Fair enough, marking this one as resolved.